tests/unit/scp_test.py

```diff
@@ -57,3 +58,39 @@ def test_get(dst_path, src_path, ssh_scp, transmit_payload):
     """Check that SCP file download works."""
     ssh_scp.get(str(src_path), str(dst_path))
     assert dst_path.read_bytes() == transmit_payload
+
+
+@pytest.fixture
+def dst_path_exists(tmp_path):
+    """Return a local path that exists."""
+    path = tmp_path / 'non-existing.txt'
+    path.write_bytes('whatever'.encode())
+    return path
+
+
+def test_get_existing_local(dst_path_exists, src_path, ssh_scp, transmit_payload):
+    """Check that SCP file download works and overrides local file if it exists."""
+    ssh_scp.get(str(src_path), str(dst_path_exists))
+    assert dst_path_exists.read_bytes() == transmit_payload
+
+
+@pytest.fixture
+def large_payload():
+    """Generate a large test payload."""
+    # buffer of random 1024 bytes -- just printable for easier debugging
+    rand = ''.join(chr(random.randint(32, 126)) for _ in range(1024))
+    return ''.join(rand for _ in range(65)).encode()
+
+
+@pytest.fixture
+def src_path_large(tmp_path, large_payload):
+    """Return a remote path that is a file larger than 64k B, which is the most libssh can read at once."""
+    path = tmp_path / 'large.txt'
+    path.write_bytes(large_payload)
+    return path
+
+
+def test_get_large(dst_path, src_path_large, ssh_scp, large_payload):
+    """Check that SCP file download works also for large files."""
```

Comment:
I think this would be a more accurate high-level description:
```suggestion
    """Check that SCP file download gets over 64kB of data."""
```